.github/dependabot.yml

```
@@ -0,0 +1,7 @@
+version: 2
+updates:
+
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
```

This could also keep website dependencies up-to-date, in `./modules/website`.
For example, Docusaurus 2.0.0-beta.14 -> 2.0.1, and React 17 -> 18.


```suggestion
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "monthly"
  - package-ecosystem: 'npm'
    directory: '/'
    schedule:
      interval: 'monthly'
```

---

let's give it a go, I don't see why not.